pkg/virt-controller/watch/vm/vm.go

```diff
@@ -2799,7 +2799,7 @@ func (c *Controller) trimDoneVolumeRequests(vm *virtv1.VirtualMachine) {
 	vm.Status.VolumeRequests = tmpVolRequests
 }
 
-func validLiveUpdateVolumes(oldVMSpec *virtv1.VirtualMachineSpec, vm *virtv1.VirtualMachine) bool {
+func validLiveUpdateVolumes(oldVMSpec virtv1.VirtualMachineSpec, vm *virtv1.VirtualMachine) bool {
```

Comment:
Why not keep the ptrs in `validLiveUpdateVolumes` and `validLiveUpdateDisks` to avoid more copies of the `lastSeenVMSpec` copy?